### ISSUE TITLE
Inception v3 failed to enqueue convolution on stream: CUDNN_STATUS_EXECUTION_FAILED

### ISSUE THREAD

**Issue by hfutxrg** [tensorflow/tensorflow#4726]
I was following the instructions in https://github.com/tensorflow/models/tree/master/inception to train inception v3. I created the TFRecord data successfully. But when I tried to train the network using the following command:
bazel-bin/inception/imagenet_train --num_gpus=4 --batch_size=128 --train_dir=... --data_dir=...
It has the following warning and error:

WARNING:tensorflow:Passing a `GraphDef` to the SummaryWriter is deprecated. Pass a `Graph` object instead, such as `sess.graph`.
F tensorflow/stream_executor/cuda/cuda_dnn.cc:1979] failed to enqueue convolution on stream: CUDNN_STATUS_EXECUTION_FAILED

### Environment info
Operating System: Redhat 7.2

Installed version of CUDA and cuDNN: 
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):
-rw-r--r-- 1 root root   560184 May  8 02:00 /usr/local/cuda-8.0/lib64/libcudadevrt.a
lrwxrwxrwx 1 root root       16 Sep 12 15:13 /usr/local/cuda-8.0/lib64/libcudart.so -> libcudart.so.8.0
lrwxrwxrwx 1 root root       19 Sep 12 15:13 /usr/local/cuda-8.0/lib64/libcudart.so.8.0 -> libcudart.so.8.0.27
-rwxr-xr-x 1 root root   394472 May  8 02:00 /usr/local/cuda-8.0/lib64/libcudart.so.8.0.27
-rw-r--r-- 1 root root   737516 May  8 02:00 /usr/local/cuda-8.0/lib64/libcudart_static.a
-rwxr-xr-x 1 root root 60696704 Sep 24 09:12 /usr/local/cuda-8.0/lib64/libcudnn.so
-rwxr-xr-x 1 root root 60696704 Sep 24 09:12 /usr/local/cuda-8.0/lib64/libcudnn.so.5
-rwxr-xr-x 1 root root 60696704 Sep 24 09:12 /usr/local/cuda-8.0/lib64/libcudnn.so.5.1.3
-rw-r--r-- 1 root root 59715990 Sep 24 09:12 /usr/local/cuda-8.0/lib64/libcudnn_static.a

If installed from source, provide 

1. The commit hash (`git rev-parse HEAD`):
f98c5ded31d7da0c2d127c28b2c16f0307a368f0
2. The output of `bazel version`
.
Build label: 0.3.1- (@non-git)
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Thu Sep 29 22:19:27 2016 (1475187567)
Build timestamp: 1475187567
Build timestamp as int: 1475187567

### Logs or other output that would be helpful
(If logs are large, please upload as attachment or provide link).
Except the above warning and error, there are the following warnings:
WARNING:tensorflow:tf.variable_op_scope(values, name, default_name) is deprecated, use tf.variable_scope(name, default_name, values)
WARNING:tensorflow:tf.op_scope(values, name, default_name) is deprecated, use tf.name_scope(name, default_name, values)

**Comment by concretevitamin** [comment]
Could you try a smaller batch size, to see if it works?

**Comment by hfutxrg** [comment]
Thanks. I tried a smaller batch size 64 and it works now.
